www/js/crypter/crypto_app.js

```diff
@@ -873,7 +864,7 @@ window.filesender.crypto_app = function () {
             oReq.addEventListener("progress", function(evt){
                 window.filesender.log("downloadAndDecryptChunk(progress) chunkid " + chunkid
                                       + " loaded " + evt.loaded + " of total " + evt.total );
-                if (evt.lengthComputable || evt.loaded) {
+                if (evt.lengthComputable) {
```

Comment:
I don't think we want to revert this change https://github.com/filesender/filesender/pull/2324